py/picca/utils.py

```diff
@@ -107,30 +110,46 @@ def smooth_cov(xi,
     sum_correlation = {}
     counts_correlation = {}
     for index in range(num_bins):
+        index_r_par = int(r_par[index]/delta_r_par)
```

Comment:
This is only necessary/used when `per_r_par=True`, right? Maybe we can add it inside an `if` bloc?